### PR TITLE
Fix Scroll top Button

### PR DESCRIPTION
**Pull request by 7ORP3DO**
### Issue : 

Fix Issue: #221 

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots

Before :

https://user-images.githubusercontent.com/80192140/150681942-c89ecabc-515a-4a9c-b52c-e3e5b5f0293c.mp4

After :

https://user-images.githubusercontent.com/80192140/150681972-03e92721-9ed7-4936-a44d-0d5adf03921d.mp4

### CODE REVIEW

File: js/scrollBtn.js [hunk 1]
```diff
@@ -1,9 +1,21 @@
...
+
 scrollToTopBtn.addEventListener('click', function () {
   scrollTo({
     top: 0,
     left: 0,
     behavior: 'smooth',
   });
 });
+
+function scrollFunction() {
```
Comment (rohan09-raj) [inline js/scrollBtn.js:15]: Can we please use es6 functions ??

File: css/main.css [hunk 2]
```diff
@@ -12,7 +12,8 @@ body {
 #scroll-to-top-btn {
   position: fixed;
   z-index: 100;
-  bottom: 20px;
+  bottom: -100px;
+  /* bottom: 20px; */
```
Comment (rohan09-raj) [inline css/main.css:16]: Can we please remove this dead code ??

**Review by rohan09-raj — CHANGES_REQUESTED** [review]

File: css/main.css (hunk 2 above)
Comment (7ORP3DO) [reply css/main.css:16]: Sure 👍

**Review by rohan09-raj — APPROVED** [review]
Thanks for the changes, good work, approving your PR 🎉 
Congrats on getting your first PR merged to Real Dev Squad

**Pull request merged**